api/src/main/java/org/openmrs/util/VelocityExceptionHandler.java

```diff
@@ -29,18 +31,19 @@
 	 * @see org.apache.velocity.app.event.MethodExceptionEventHandler#methodException(java.lang.Class,
 	 *      java.lang.String, java.lang.Exception)
 	 */
+	
 	@Override
-	public Object methodException(Class claz, String method, Exception e) throws Exception {
-		
+	public Object methodException(Context context, Class claz, String method, Exception e, Info info) {
```

Comment:
Can you remind me why we needed to make this change?

Reply:
context was a parameter in the method signature of methodException method which VelocityExceptionHandler was overriding as a result of implementing the MethodExceptionEventHandle.
https://velocity.apache.org/engine/2.0/apidocs/org/apache/velocity/app/event/MethodExceptionEventHandler.html